src/main/java/Jarvas.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Scanner;
+public class Jarvas {
+    public static void initialiseBot(){
+        System.out.println(" _____                                  ");
+        System.out.println("(___  )                                 ");
+        System.out.println("    | |   _ _  _ __  _   _    _ _   ___ ");
+        System.out.println(" _  | | /'_` )( '__)( ) ( ) /'_` )/',__)");
+        System.out.println("( )_| |( (_| || |   | \\_/ |( (_| |\\__, \\");
+        System.out.println("`\\___/'`\\__,_)(_)   `\\___/'`\\__,_)(____/");
+
+        Reply.printReply("Hello! I'm Jarvas", "What can I do for you?");
+    }
+    public static void terminateBot(){
+        Reply.printReply("Bye, see you soon!");
+    }
+    public static void runTime(){
+        Scanner in = new Scanner(System.in);
+        String userInput = in.nextLine();
+        Task[] tasks = new Task[100];
+
+        while (true) {
```

Comment:
Is the logic of the loop clear enough to specify an explicit exit condition? Something to the effect of while instruction != "bye".